winui/Rating/Precision.md

```diff
@@ -37,7 +37,7 @@ rating.ItemsCount = 5;
 
 ## Half
 
-Rating value can be provided with half precision.
+Rating value can be provided with nearest half value.
```

Comment:
The rating value can be provided with the nearest half value.

Reply:
Changed the content as mentioned above.